### PR TITLE
Remove goog.labs.userAgent

### PR DESCRIPTION
**Pull request by nicholas-l**
Not sure if there is a cleaner way of doing this so comments welcomed :)

### CODE REVIEW

File: src/ol/reproj/reproj.js [hunk 1]
```diff
@@ -18,8 +20,9 @@ goog.require('ol.proj');
  * @type {boolean}
  * @private
  */
-ol.reproj.browserAntialiasesClip_ = !goog.labs.userAgent.browser.isChrome() ||
-                                    goog.labs.userAgent.platform.isIos();
+ol.reproj.browserAntialiasesClip_ = !(!winNav.userAgent.match('CriOS') && window.chrome !== null &&
+    window.chrome !== undefined && winNav.vendor === 'Google Inc.' &&
+    isOpera == false && isIEedge == false);
```
Comment (tschaub) [inline src/ol/reproj/reproj.js]: You could use an immediately invoked function expression here.  So, something like:

```js
ol.reproj. browserAntialiasesClip_ = (function() {
  var winNav = window.navigator;
  var isOpera = winNav.userAgent.indexOf('OPR') > -1;
  var isIEedge = winNav.userAgent.indexOf('Edge') > -1;
  return // ...
}
```

**Comment by nicholas-l** [comment]
@tschaub Changed to a function

**Comment by nicholas-l** [comment]
Anyone able to review?

File: src/ol/reproj/reproj.js [hunk 2]
```diff
@@ -18,8 +16,18 @@ goog.require('ol.proj');
...
+  var isOpera = winNav.userAgent.indexOf('OPR') > -1;
+  var isIEedge = winNav.userAgent.indexOf('Edge') > -1;
+  return !(
+    !winNav.userAgent.match('CriOS') &&  // Not Chrome on iOS
+    winChrome !== null && winChrome !== undefined && // Has chrome in window
+    winNav.vendor === 'Google Inc.' && // Vendor is Google.
+    isOpera == false && // Not Opera
+    isIEedge == false // Not Edge
+  );
+})(window.navigator, window.chrome)
```
Comment (ahocevar) [inline src/ol/reproj/reproj.js]: I'd use `goog.global` instead of `window` here, so this does not fail in non-browser environments.

**Comment by ahocevar** [comment]
I added one more comment @nicholas-l. When you're able to address it, we can merge this. Thanks!

File: src/ol/reproj/reproj.js (hunk 2 above)
Comment (nicholas-l) [inline src/ol/reproj/reproj.js]: What about ```this```? goog.global = this in http://google.github.io/closure-library/api/source/closure/goog/base.js.src.html#l48

**Comment by nicholas-l** [comment]
@ahocevar I have changed the global to ```this``` as I didn't want to add another goog :).

**Comment by ahocevar** [comment]
Are you sure that `this` will also work in a browserify context? We use `goog.global` for now everywhere else.

**Comment by nicholas-l** [comment]
I believe it would work as in this is what goog.global is doing, maybe we should have an ol.global? http://google.github.io/closure-library/api/source/closure/goog/base.js.src.html#l48

**Comment by ahocevar** [comment]
`ol.global` would be a next step. This is why I'd prefer to use `goog.global` for now, because it will be easier and less error prone to change to `ol.global` later.

**Comment by nicholas-l** [comment]
ok sounds good

**Comment by ahocevar** [comment]
Thanks @nicholas-l.

**Comment by nicholas-l** [comment]
Have created #5178 to tackle global

**Pull request merged**